beacon-chain/sync/initial-sync/blocks_fetcher_utils.go

```diff
@@ -382,33 +380,6 @@ func (f *blocksFetcher) calculateHeadAndTargetEpochs() (headEpoch, targetEpoch p
 	return headEpoch, targetEpoch, peers
 }
 
-// custodyGroupsFromPeer compute all the custody groups indexed by peer.
-func (f *blocksFetcher) custodyGroupsFromPeer(peers map[peer.ID]bool) (map[peer.ID]map[uint64]bool, error) {
```

Comment:
These are linter spotted issues, it's not used anymore, however not sure why it's not caught by CI in https://github.com/prysmaticlabs/prysm/pull/14977/files#diff-c81d6a0f926408e8bfafff8493bd30f16f486afd6d38b9582a8255c90c464f04